pkg/rancher-desktop/backend/lima.ts

```diff
@@ -48,6 +49,8 @@ import { jsonStringifyWithWhiteSpace } from '@pkg/utils/stringify';
 import { defined, RecursivePartial } from '@pkg/utils/typeUtils';
 import { openSudoPrompt } from '@pkg/window';
 
+/* eslint @typescript-eslint/switch-exhaustiveness-check: "error" */
```

Comment:
Should we add this globally? I'm not thrilled that a `default:` deals with it, but it still seems useful in general.

Reply:
It would be useful, but I wanted to start doing it piecemeal in case we hit things we imported from dashboard…